app/population/controllers/dashboard.js

```diff
@@ -0,0 +1,18 @@
+function dashboard(req, res) {
+  const today = new Date().toISOString()
+
+  const { context, dateRange, pagination, resultsAsPopulationTable } = req
+  const { period } = req.params
+
+  res.render('population/view/dashboard', {
+    context,
+    pageTitle: 'dashboard::page_title',
+    pagination,
+    resultsAsPopulationTable,
+    period,
+    currentWeek: dateRange,
+    today,
```

Comment:
If this is required for the view you can use the global value that is set in the [locals middleware](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/3ed4f72b57bee7dbae39b6572b73c05a280048d0/common/middleware/locals.js#L10).

Reply:
Apparently it's the only one that wasn't required - so I've removed it.